composer.json

```diff
@@ -22,7 +22,7 @@
     "require": {
         "sentry/sentry": "^2.3",
```

Comment:
Since we need to reimplement the new timeouts, I would say that we need to require the corresponding patch here to make it all work.

Reply:
Good point 👌